app/assets/stylesheets/darkswarm/_shop-inputs.css.scss

```diff
@@ -27,22 +23,22 @@
       border-color: #b3b3b3;
       text-align: right;
 
-      @media all and (max-width: 1024px) {
+      @include breakpoint(desktop) {
```

Comment:
this is very nice,
This deserves to be a standard with an entry here: https://github.com/openfoodfoundation/openfoodnetwork/wiki/Code-Conventions#css